src/cordova/restore-util.js

```diff
@@ -108,7 +108,7 @@ function installPlatformsFromConfigXML (platforms, opts) {
 
     const platformInfo = platformIDs.map(plID => ({
         name: plID,
-        spec: specs[plID]
+        spec: specs[`cordova-${plID}`]
```

Comment:
The following should approximate the logic we use in `addHelper.js` to determine the platform spec:

```suggestion
        spec: specs[`cordova-${plID}`] || specs[plID]
```